.github/workflows/release.yml

```
@@ -93,3 +106,31 @@ jobs:
       env:
         FLIT_USERNAME: __token__
         FLIT_PASSWORD: ${{ secrets.PYPI_KEY }}
+        FLIT_INDEX_URL: https://pypi.org/
+
+  publish-testpypi:
+    if: github.event_name == 'pull_request' && contains(fromJson(toJson(github.event.pull_request.labels)).*.name, 'test-release')
+
+    name: Publish to test PyPI
+
+    needs: [pre-commit, tests]
```

If you're publishing on TestPyPI, you don't really care about the tests right?

```suggestion
    needs: [pre-commit]
```

---

I want to have the same workflow run for testpypi release as for pypi release, so the official release pipeline does not fail up because something was not considered.